solver/src/driver.rs

```diff
@@ -423,7 +423,8 @@ impl Driver {
             let mut settlements = match settlements {
                 Ok(settlement) => settlement,
                 Err(err) => {
-                    tracing::error!("solver {} error: {:?}", name, err);
+                    self.metrics.solver_time_out(name);
```

Comment:
Note that this is not just a timeout, it can also be an actual error. For example HTTP errors in the HTTP solver also hit this path.